### ISSUE TITLE
Creating layers in a loop

### ISSUE THREAD

**Issue by asheetal** [rstudio/keras#951]
I am building a multi-label keras model in R where the number of outputs is stored in an integer. My goal is to create the last layer in a for-loop somethign like this. 

>for (i in 1:number_output_vars){
>  model_keras[i] <- model_keras_pre %>%
>    layer_dense(units = 1, kernel_initializer = "zeros") %>%
>    layer_activation("relu")
>}

However, the above code generates an error 

> Error in model_keras[i] <- model_keras_pre %>% layer_dense(units = 1,  : 
>  object of type 'closure' is not subsettable

I want to have different loss function for different labels, hence need some kind of for-loop to create the last layer. Is there a better way to create layers in a for-loop?

**Comment by dfalbel** [comment]
Can you share a reproducible example? Creating layers from a loop should just work, but I'm not sure what's `model_keras` in your example.

**Comment by asheetal** [comment]
> Can you share a reproducible example? Creating layers from a loop should just work, but I'm not sure what's `model_keras` in your example.

Before the loop I tried with model_keras <- vector()
Did not work

I am hoping someone who created a layer in a loop could point me to method. I will then follow that style
